SU-HSDO/suhumsci
# shs-95: this fixes a couple bugs related to video embeds

## Summary
This fixes a couple bugs related to audio embeds.

## Steps to Test
1. Create a flexible page and and add a text area component. Embed a YouTube video.
2. Hover over the video and click edit media. Switch between left and right and ensure the video still displays in the WYSIWYG (screenshot below)
3. Settle on 'right' aligned. Save the page and view it. Ensure that you can see the video and play it.

## Screenshots
![Screen Shot 2022-12-21 at 9 12 46 AM](https://user-images.githubusercontent.com/8405274/208925924-ef3909ae-1083-4cb1-a1f7-d64f2458e4f4.png)

![Screen Shot 2022-12-21 at 9 12 33 AM](https://user-images.githubusercontent.com/8405274/208925938-45c9693e-2b0c-4173-907a-b9c05b278fbd.png)
